src/main/java/com/treasure/hunt/game/GameEngine.java

```diff
@@ -30,37 +28,81 @@
     /**
      * The height of the playing area.
      */
-    public static final int HEIGHT = 200;
+    @Getter
+    protected final int height;
     /**
      * The width of the playing area.
      */
-    public static final int WIDTH = 200;
+    @Getter
+    protected final int width;
```

Comment:
We need to tell the hider and searcher but this value

Reply:
This is right, I'm going to fix this